ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
```

Comment:
@pvnr0082t I chose to use "WhitestormJS" instead of "whs" because the author's request (#10616) is to add "WhitestormJS", I am not sure which one is a better choice? Should we follow the author's request or package.json/bower.json?